submissions/nhunzaker/lib/server.js

```diff
@@ -0,0 +1,28 @@
+var Hapi = require('hapi')
+var pkg = require('../package')
+
+exports.metadata = {
```

Comment:
I like that you've associated the version with this file. It means pulling less dependencies directly into the tests. e.g. my test file imports package.json directly to test the version response.